src/frontend/js/components/CourseRunEnrollment/index.spec.tsx

```diff
@@ -177,6 +180,38 @@ describe('<CourseRunEnrollment />', () => {
     screen.getByText('You are enrolled in this course run');
   });
 
+  it("doesn't shows a link to the course if the user is already enrolled and if the course hasn't started yet", async () => {
+    const username = initializeUser();
+    const courseRun: CourseRun = factories.CourseRunFactory.generate();
+    courseRun.state.priority = 0;
+    courseRun.start = String(faker.date.future().toISOString());
+
+    const courseRunDeferred = new Deferred();
+    fetchMock.get(`/api/v1.0/course-runs/${courseRun.id}/`, courseRunDeferred.promise);
+    const enrollmentsDeferred = new Deferred();
+    fetchMock.get(
+      `${endpoint}/api/enrollment/v1/enrollment/${username},${courseRun.resource_link}`,
+      enrollmentsDeferred.promise,
+    );
+
+    render(
+      <IntlProvider locale="en">
+        <SessionProvider>
+          <CourseRunEnrollment context={contextProps} courseRun={getCourseRunProp(courseRun)} />
+        </SessionProvider>
+      </IntlProvider>,
+    );
+    screen.getByRole('status', { name: 'Loading enrollment information...' });
+
+    await act(async () => {
+      courseRunDeferred.resolve(courseRun);
+      enrollmentsDeferred.resolve({ is_active: true });
+    });
+
+    expect(screen.queryByRole('link', { name: 'Go to course' })).toBeNull();
+    screen.getByText('You are enrolled in this course run');
```

Comment:
In terms of Ux, do we need to precise that we are now waiting for the course to open or will the user understand?

Reply:
Just above the hidden button, start date is already displayed.
I can't post a screenshot right now, but I'll do it tomorrow.
I think it's understandable without additional message.

Reply:
![image](https://user-images.githubusercontent.com/720491/106723896-dfc58480-6607-11eb-97ea-55b2e78f7e5d.png)
Currently, it looks likes this

Reply:
oh ok! Then I think we should provide a link on this phrase that points to the user's dashboard (see profile menu) so that he/she can go see the courses to which he/she is enrolled. WDYT?

Reply:
Good for me, any thought @jbpenrath ?

Reply:
Yep I agree with @sampaccoud, good idea to allow user to access to its dashboard and see its enrollments.

Reply:
In the menu, I see a ink like this `https://edx.local.dev:8073/u/edx` but, you want this one `https://edx.local.dev:8073/dashboard` , right ?